### DISCUSSION TITLE
Search when sharing publicly

### DISCUSSION THREAD

**Discussion by yajo** [outline/outline#3003] [Ideas]
If you share a document (and maybe subdocuments) publicly, the public users cannot search. Search is a powerful feature in any knowledge base, so it should be available for that shared document and children.

**Comment by tommoor** [comment]
I created an issue to track this, the maintainers also talked about it recently.

**Comment by jenningsb2** [comment]
Came here to add this! This is a critical feature when hosting larger documentation repos.

**Comment by tommoor** [comment]
Hi, work is ongoing – the tracking issue is here: https://github.com/outline/outline/issues/3004
